examples/azure_flow_logs_storage_accounts/versions.tf

```
@@ -0,0 +1,12 @@
+terraform {
+  required_providers {
+    illumio-cloudsecure = {
+      source  = "illumio/illumio-cloudsecure"
+      version = ">= 1.1.0"
```

The `azure_flow_logs_storage_account` resource got added in 1.2.0.
```suggestion
      version = ">= 1.2.0"
```